ensime_shared/client.py

```diff
@@ -85,164 +88,144 @@ def setup_logger():
 
         super(EnsimeClient, self).__init__()
         self.editor = editor
-        self.launcher = launcher
+        self.config = config
 
         self.log = setup_logger()
         self.log.debug('__init__: in')
         self.editor.initialize()
 
         self.ws = None
-        self.ensime = None
```

Comment:
I think that `:EnStatus` would be a better name and it's better to tackle this in another PR.